x-pack/plugins/session_view/public/components/detail_panel_description_list/styles.ts

```diff
@@ -17,8 +17,22 @@ export const useStyles = () => {
       padding: euiTheme.size.s,
     };
 
+    const tabListTitle = {
```

Comment:
if i initialize it as `CSSObject`, `titleProps` and `descriptionProps` complains that they are not compatible